Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -56,7 +56,52 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
           Please make sure to read and understand the documentation page on :ref:`taint effects and unmanaged pods<taint_effects>`.
 
-    .. group-tab:: AKS
+    .. group-tab:: AKS (BYOCNI)
+
+       .. note::
+
+          BYOCNI is the preferred way to run Cilium on AKS, however integration
+          with the Azure stack via the :ref:`Azure IPAM<ipam_azure>` is not
+          available. If you require Azure IPAM, refer to the AKS (Azure IPAM)
+          installation.
+
+       The following commands create a Kubernetes cluster using `Azure
+       Kubernetes Service <https://docs.microsoft.com/en-us/azure/aks/>`_ with
+       no CNI plugin pre-installed (BYOCNI). See `Azure Cloud CLI
+       <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+       for instructions on how to install ``az`` and prepare your account, and
+       the `Bring your own CNI documentation
+       <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_
+       for more details about BYOCNI prerequisites / implications.
+
+       .. note::
+
+          BYOCNI requires the ``aks-preview`` CLI extension with version >=
+          0.5.55, which itself requires an ``az`` CLI version >= 2.32.0 .
+
+       .. code-block:: bash
+
+           export NAME="$(whoami)-$RANDOM"
+           export AZURE_RESOURCE_GROUP="${NAME}-group"
```

Comment:
Are the `export`s needed?

Reply:
They are not needed, I just copy/pasted whatever we had in the already existing AKS instructions. I would suggest to do a separate PR to remove all unneeded `export` at once from all GSG if we want to do so, this way we keep consistency in the docs before and after that PR is merged.